Library/ViewModels/PledgeAmountViewModelTests.swift

```diff
@@ -122,19 +122,19 @@ internal final class PledgeAmountViewModelTests: TestCase {
     self.doneButtonIsEnabled.assertValues([true])
 
     self.vm.inputs.stepperValueChanged(2)
-    self.doneButtonIsEnabled.assertValues([true, true])
```

Comment:
Mostly `skipRepeats` for these signals...should have done that before, only noticed now.